packages/next/src/client/components/react-dev-overlay/internal/helpers/launchEditor.ts

```diff
@@ -32,6 +32,7 @@ function isTerminalEditor(editor: string) {
   switch (editor) {
     case 'vi':
     case 'vim':
+    case 'nvim':
```

Comment:
also need to add to `COMMON_EDITORS_LINUX`?

Reply:
thank you for reviewing.
indeed. I'll add commit.

Reply:
pushed vercel/next.js@f6d5661c5d